src/mod/irc.mod/tclirc.c

```diff
@@ -434,6 +434,30 @@ static int tcl_ischaninvite STDVAR
   return TCL_OK;
 }
 
+/* Checks internal tracking of IRC server away status, as updated by IRC
+ * server 352 and AWAY messages. Meant mostly for use with the IRCv3 away-notify
+ * capability, it may not be accurate using only 352s.
+ */
+static int tcl_isaway STDVAR
+{
+  struct chanset_t *chan;
+  memberlist *m;
+
+  BADARGS(2, 2, " nick");
+
+  for (chan = chanset; chan; chan = chan->next) {
+    m = ismember(chan, argv[1]);     /* In my channel list copy? */
+    if (m) {
+        if chan_ircaway(m) {
```

Comment:
where are the paranthesis around the if-condition? the indentation is weird, too